Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -129,11 +129,6 @@ fetch() {
     curl_args[${#curl_args[*]}]="--progress-bar"
   fi
 
-  if [[ "${HOMEBREW_MACOS_VERSION_NUMERIC}" -lt "100600" ]]
```

Comment:
This reference to `HOMEBREW_MACOS_VERSION_NUMERIC` can also be removed:

https://github.com/Homebrew/brew/blob/3a5b17a421fea58241efd9398349eb17c5e92277/Library/Homebrew/cmd/vendor-install.sh#L5